hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/footer/AbstractDeleteActionsLayout.java

```diff
@@ -288,6 +282,43 @@ protected void hideBulkUploadStatusButton() {
     }
 
     /**
+     * Only in deployment view count message is displayed.
```

Comment:
How do _AbstractDeleteActionsLayout_ know that count message is only displayed in deployment view?